spacy/scorer.py

```diff
@@ -37,7 +37,7 @@ def fscore(self):
 class Scorer(object):
     def __init__(self, eval_punct=False):
         self.tokens = PRFScore()
-        self.sbd = PRFScore()
+        self.sentencizer = PRFScore()
```

Comment:
This is internals only and doesn't really refer to the component, so I think this should stay:

```suggestion
        self.sbd = PRFScore()
```